<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PolarWidget</class>
 <widget class="QWidget" name="PolarWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>680</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBoxLearn">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>510</y>
     <width>231</width>
     <height>161</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="title">
    <string>采样窗口范围</string>
   </property>
   <widget class="QPushButton" name="btnAddSamplingAngle">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>110</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSubSamplingAngle">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>110</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonSamplingAngelStart">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>41</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>起始角度(A1)</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonSamplingAngelEnd">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>79</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>终止角度(A2)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtSamplingAngelStart">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>34</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtSamplingAngelEnd">
    <property name="geometry">
     <rect>
      <x>131</x>
      <y>70</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_location">
   <property name="geometry">
    <rect>
     <x>667</x>
     <y>50</y>
     <width>341</width>
     <height>128</height>
    </rect>
   </property>
   <property name="title">
    <string>搜索范围</string>
   </property>
   <widget class="QRadioButton" name="radioButton_Global">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>64</y>
      <width>87</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>全局搜索</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_Part">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>64</y>
      <width>87</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>局部搜索</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBoxSearch">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>510</y>
     <width>231</width>
     <height>161</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="title">
    <string>搜索窗口范围</string>
   </property>
   <widget class="QPushButton" name="btnAddSearchAngle">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>110</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSubSearchAngle">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>110</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonSearchAngelEnd">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>79</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>终止角度(B2)</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonSearchAngelStart">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>41</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>起始角度(B1)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtSearchAngelStart">
    <property name="geometry">
     <rect>
      <x>127</x>
      <y>33</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtSearchAngelEnd">
    <property name="geometry">
     <rect>
      <x>129</x>
      <y>72</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_Angle">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>190</y>
     <width>341</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>角度精度</string>
   </property>
   <widget class="QRadioButton" name="radioButton_Hight">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>57</y>
      <width>53</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0.1°</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_Mid">
    <property name="geometry">
     <rect>
      <x>145</x>
      <y>57</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0.5°</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_Low">
    <property name="geometry">
     <rect>
      <x>37</x>
      <y>57</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>1°</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBoxSimilarity">
   <property name="geometry">
    <rect>
     <x>671</x>
     <y>309</y>
     <width>341</width>
     <height>100</height>
    </rect>
   </property>
   <property name="title">
    <string>相似度</string>
   </property>
   <widget class="QLabel" name="LabelSimilarlity">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>29</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 10pt &quot;微软雅黑&quot;;</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="SimilarlityDownBtn">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>41</y>
      <width>50</width>
      <height>40</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>50</width>
      <height>29</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>50</width>
      <height>40</height>
     </size>
    </property>
    <property name="text">
     <string>-</string>
    </property>
    <property name="autoRepeat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="SimilarlityUpBtn">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>41</y>
      <width>50</width>
      <height>40</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>50</width>
      <height>29</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>50</width>
      <height>40</height>
     </size>
    </property>
    <property name="text">
     <string>+</string>
    </property>
    <property name="autoRepeat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QSlider" name="SimilarlitySlider">
    <property name="geometry">
     <rect>
      <x>73</x>
      <y>54</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>152</width>
      <height>19</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>450</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="pageStep">
     <number>1</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBoxRIn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>510</y>
     <width>251</width>
     <height>161</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="title">
    <string>内圆半径调整</string>
   </property>
   <widget class="QPushButton" name="btnAddRadiusIn">
    <property name="geometry">
     <rect>
      <x>71</x>
      <y>94</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtRadiusIN">
    <property name="geometry">
     <rect>
      <x>108</x>
      <y>51</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSubRadiusIn">
    <property name="geometry">
     <rect>
      <x>131</x>
      <y>94</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_29">
    <property name="geometry">
     <rect>
      <x>59</x>
      <y>55</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>半径:</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBoxROut">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>510</y>
     <width>251</width>
     <height>161</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="title">
    <string>外圆半径调整</string>
   </property>
   <widget class="QPushButton" name="btnAddRadiusOut">
    <property name="geometry">
     <rect>
      <x>75</x>
      <y>94</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtRadiusOut">
    <property name="geometry">
     <rect>
      <x>112</x>
      <y>51</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSubRadiusOut">
    <property name="geometry">
     <rect>
      <x>135</x>
      <y>94</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_30">
    <property name="geometry">
     <rect>
      <x>62</x>
      <y>54</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>半径:</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
